src/components/header/Header.jsx

```diff
@@ -34,14 +36,18 @@ const Header = () => {
 					<Link to="/wishlist" className="link-no-style">
 						<button className="btn btn-primary btn-icon btn-hide">
 							<i className="far fa-heart fa-2x badge-parent">
-								<span className="badge-content badge-itself">5</span>
+								<span className="badge-content badge-itself">
+									{productState.wishlist.length}
+								</span>
 							</i>
 						</button>
 					</Link>
 					<Link to="/cart" className="link-no-style">
 						<button className="btn btn-primary btn-icon btn-hide">
 							<i className="fas fa-shopping-basket fa-2x badge-parent">
-								<span className="badge-content badge-itself">3</span>
+								<span className="badge-content badge-itself">
+									{productState.cart.length}
+								</span>
 							</i>
```

Comment:
just a suggestion you can use NavLink here instead of Link

Reply:
used navLink instead Link in commit.